content/en/docs/ops/ambient/upgrade/helm-upgrade/index.md

```diff
@@ -83,12 +85,12 @@ $ helm upgrade ztunnel istio/ztunnel -n istio-system
 
 ### Upgrade the CNI Component
 
-The Istio CNI agent is responsible for detecting the pods that belong to the ambient mesh, and configuring the traffic redirection between pods and the ztunnel DaemonSet. It is not part of the data plane or control plane.
+The Istio CNI agent is responsible for detecting the pods that belong in the ambient mesh, informing ztunnel that proxy ports should be established within added pods, and configuring traffic redirection within the pod network namespace. It is not part of the data plane or control plane.
 
 The Istio CNI agent at version 1.x is compatible with control plane at version 1.x-1, 1.x, and 1.x+1, which means the Istio CNI agent and the Istio control plane can be upgraded independently and in any order, as long as their version difference is within one minor version.
 
 {{< warning >}}
-Upgrading the Istio CNI agent will reconfigure networking on the node, and as such will momentarily disrupt node traffic. To manage blast radius to application pods during Istio CNI agent upgrade, node cordons are recommended.
+Upgrading the Istio CNI agent to a compatible version in-place will not disrupt networking for running pods already successfully added to ambient mesh, but new pods will fail to schedule on the node until the upgrade is complete and the upgraded Istio CNI agent on the node passes readiness checks. If this is a significant disruption concern, or stricter blast radius controls are desired for CNI upgrades, node taints and/or node cordons are recommended.
```

Comment:
+1 on this important clarification

Reply:
may be worth mention this will impact recovered/restarted pods as well, i think?

Reply:
Echo Lin's comment about recovered/restarted pods prior to CNI passing readiness checks.. but maybe expand that once the readiness checks are complete, pods will be added/removed from ambient mesh as appropriate.

Reply:
I'll think about how to word this better. "New pod" by definition covers any new pod on the node, whether it comes from a new resource being deployed, or a restart of pods belonging to an existing resource - it's still scheduling a new pod on the node.

Reply:
I took a stab at it, LMK if it's worse or better.